packages/accordion/src/AccordionItem.ts

```diff
@@ -68,6 +107,10 @@ export class AccordionItem extends Focusable {
         }
     }
 
+    protected renderChevronIcon = (): TemplateResult => {
+        return chevronIcon[this.size]();
```

Comment:
If we can have no size (because we stop passing it from the parent), then we likely need a fallback here.
```suggestion
        return chevronIcon[this.size || 'm']();
```